### PR TITLE
Move to keraslayers fasterrcnn fpn keras feature extractor

### PR DESCRIPTION
**Pull request by syiming**
# Description

> :memo: Return keras layers instead of keras models for fasterrcnn fpn keras feature extractor
>  
> * Please also include relevant motivation and context.  
> * List any dependencies that are required for this change.  

## Type of change

For a new feature or function, please create an issue first to discuss it
with us before submitting a pull request.

Note: Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation update
- [ ] TensorFlow 2 migration
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] A new research paper code implementation
- [x] Other (Fix 'unknow graph' error)

## Tests

> :memo: Please describe the tests that you ran to verify your changes.
>  
> * Provide instructions so we can reproduce.  
> * Please also list any relevant details for your test configuration.  

**Test Configuration**:

## Checklist

- [x] I have signed the [Contributor License Agreement](https://github.com/tensorflow/models/wiki/Contributor-License-Agreements).
- [x] I have read [guidelines for pull request](https://github.com/tensorflow/models/wiki/Submitting-a-pull-request).
- [x] My code follows the [coding guidelines](https://github.com/tensorflow/models/wiki/Coding-guidelines).
- [x] I have performed a self [code review](https://github.com/tensorflow/models/wiki/Code-review) of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

File: research/object_detection/models/faster_rcnn_resnet_v1_fpn_keras_feature_extractor.py [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+        [batch_size * self.max_num_proposals, 1024]
+        representing box classifier features for each proposal.
+    """
+    with tf.name_scope(name):
+      with tf.name_scope('ResnetV1FPN'):
+        # TODO: Add a batchnorm layer between two fc layers.
+        feature_extractor_model = tf.keras.models.Sequential([
+            tf.keras.layers.Flatten(),
+            tf.keras.layers.Dense(units=1024, activation='relu'),
+            tf.keras.layers.Dense(units=1024, activation='relu'),
```
Comment (tombstone) [inline research/object_detection/models/faster_rcnn_resnet_v1_fpn_keras_feature_extractor.py:298]: add batchnorm layer with `self._conv_hyperparams.build_batch_norm(..)

File: research/object_detection/models/faster_rcnn_resnet_v1_fpn_keras_feature_extractor.py [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+
+    Returns:
+      A Keras model that takes proposal_feature_maps:
+        A 4-D float tensor with shape
+        [batch_size * self.max_num_proposals, crop_height, crop_width, depth]
+        representing the feature map cropped to each proposal.
+
+      And returns proposal_classifier_features:
+        A 4-D float tensor with shape
+        [batch_size * self.max_num_proposals, 1024]
```
Comment (tombstone) [inline research/object_detection/models/faster_rcnn_resnet_v1_fpn_keras_feature_extractor.py:289]: correct output shape is `[batch_size * self.max_num_proposals, 1, 1, 1024]` right ?
